### PR TITLE
OG-566 Hub penalize fix

### PR DESCRIPTION
**Pull request by shahafn**

### CODE REVIEW

**Review by drortirosh — APPROVED** [review]

**Review by drortirosh — CHANGES_REQUESTED** [review]
the hub penalization fix looks OK, but it depends on a complex boosting issue, which looks much more complex (and unrelated..)

**Comment by shahafn** [comment]
> the hub penalization fix looks OK, but it depends on a complex boosting issue, which looks much more complex (and unrelated..)

Yes, once we merge the previous PR, I'll rebase this one and you'll see only the diff.

**Review by drortirosh — APPROVED** [review]

**Review by drortirosh — CHANGES_REQUESTED** [review]
missing version change

**Comment by shahafn** [comment]
> missing version change

Do you mean contract version?

File: packages/provider/src/GSNConfigurator.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const defaultGsnConfig: GSNConfig = {
   sliceSize: 3,
   relayTimeoutGrace: DEFAULT_RELAY_TIMEOUT_GRACE_SEC,
   methodSuffix: '',
-  requiredVersionRange: '2.2.0',
+  requiredVersionRange: '2.2.3',
```
Comment (drortirosh) [inline packages/provider/src/GSNConfigurator.ts:40]: this is the one version we should not update: you basically say that you don't want to run against contracts with older version.

**Review by drortirosh — APPROVED** [review]

**Pull request merged**